relayer/spy_relayer/src/helpers/redisHelper.ts

```diff
@@ -428,3 +385,51 @@ export async function monitorRedis(metrics: PromHelper) {
     await sleep(TEN_SECONDS);
   }
 }
+
+/** Check to see if a queue is in the listener memory queue before redis */
+export async function checkQueue(key: string): Promise<string | null> {
+  try {
+    const backupQueue = getBackupQueue();
+    const queuedRecord = backupQueue.find((record) => {
```

Comment:
i think you need to either remove the `{}` here or explicitly put `return` or else in this current implementation this will always return undefined

https://replit.com/@swimricky/findarrowfn?v=1

Reply:
Fixed in ae9e2fb7b5bf5ab5aeda27928dc9a22785ec9da5, keep them coming :)